hnginternship5/ziki
# #[STORY ID] Created About-us page, fix logo on subscribers & subscription page

#### What does this PR do?
* It creates the About-us page
* it shows the logo for Subscribers and subscriptions page
* Adjusted font on the contact-us page

#### Description of Task to be completed?
*About page `about.html` is created
*Subscribers page `subscribers.html` is edited
*Subscriptions page `subscriptions.html` is edited
*Contact-us  page `contact-us.html` is edited
* Styling is added to `"inline"`

#### How should this be manually tested?
* Open the site on your browser
* Type localhost/about
* You should see the about page
* Type localhost/subscribers
* You should see the subscribers page
* Type localhost/subscriptions
* You should see the subscription page
.* Type localhost/Contact-us
* You should see the Contact page

#### Screenshots (if appropriate)
About-us Page
![Screen Shot 2019-04-21 at 11 51 14 AM](https://user-images.githubusercontent.com/38491485/56469031-2397cb80-642c-11e9-8cb9-26b3552e8331.png)

Contact-us Page
![Screen Shot 2019-04-21 at 11 52 46 AM](https://user-images.githubusercontent.com/38491485/56469028-14b11900-642c-11e9-906e-10b165969e5c.png)
Logo -Subscription and subscribers page
![Screen Shot 2019-04-21 at 11 55 03 AM](https://user-images.githubusercontent.com/38491485/56469046-5641c400-642c-11e9-99f7-07f34c1672e1.png)

and this
![Screen Shot 2019-04-21 at 11 55 51 AM](https://user-images.githubusercontent.com/38491485/56469051-783b4680-642c-11e9-809b-d838ed8175b0.png)

#### Questions/Comments:

***************
* Note * edited the route.php to add /about but it hasn't been linked with the sidebar
*************
